### ISSUE TITLE
Changing hotkey to attach/detach to a cart doesn't apply to the UI text on hover.

### ISSUE THREAD

**Issue by jaydee0004** [TastyChickenLegs/BetterCarts#1]
Hi Tasty,

Changing hotkey to attach/detach to a cart doesn't apply to the UI text on hover. For example, I set it to `RightAlt`, the hover text still shows V.

I even set `allowOutOfPlaceAttach = false` and I can still see the hover text even from far away the cart.

**Comment by TastyChickenLegs** [comment]
Yes that is probably a thing.  I think I left it hardcoded while testing.  I will fix that and release a new version tonight.  Thanks for letting me know.
